salt/kubelet/node_labels.jinja

```diff
@@ -0,0 +1 @@
+KUBELET_NODE_LABELS="--node-labels={{ node_labels|join(",") }}"
```

Comment:
It may be possible to use a systemd drop in unit rather than replace the whole file / edit the package? Combined with something like this? https://gist.github.com/nickjacob/9909574

I really should not have been surprised systemd doesn't "execute" the script the same as other init systems have the past.. Oh well.

Either way, even though this is only 1 small detail of this patch, the pattern will be needed for etcd anyway, as restarting etcd services is a much bigger deal.